xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -35,11 +35,15 @@
         <Parameter Name="allowMultipleRootLevelValues" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="data">To be added.</param>
-        <param name="conformanceMode">To be added.</param>
-        <param name="allowMultipleRootLevelValues">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="data">The CBOR encoded data to read.</param>
+        <param name="conformanceMode">Specifies a conformance mode guiding the checks performed on the encoded data.
+              Defaults to <see cref="F:System.Formats.Cbor.CborConformanceMode.Strict" /> conformance mode.</param>
+        <param name="allowMultipleRootLevelValues">Specify if multiple root-level values are to be supported by the reader.
+              When set to <see langword="false" />, the reader will throw an <see cref="T:System.InvalidOperationException" /> if trying to read beyond the scope of one root-level CBOR data item.</param>
```

Comment:
Same boilerplate suggestion here:

```suggestion
        <param name="allowMultipleRootLevelValues"><see langword="true" /> to indicate that multiple root-level values are supported by the reader; otherwise, false.</param>
```